tests/utils/fastify.ts

```
@@ -0,0 +1,67 @@
+  type RouteHandlerMethod,
+  type FastifyRequest,
+} from 'fastify'
+
+import type { FastifyCoolDownProps } from '../../src/fastify/types'
+import { fastifyCoolDown } from '../../src'
+import { sleep } from './general'
+
+export const REQUEST_DELAY = 10
+
+export const getApp = async (props?: FastifyCoolDownProps) => {
+  let assertionError: { error?: Error | undefined } = {}
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'assertionError' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)